### ISSUE TITLE
RangeError

### ISSUE THREAD

**Issue by Stetocasa** [bwp91/homebridge-ewelink#273]
### What issue do you have? Please be as thorough and explicit as possible.

Since yesterday the plug-in is broke

### Details of your setup.
* Do you use (1) Homebridge (+ config-ui), (2) Homebridge (CLI) or (3) HOOBS? 
**Hoobs**

* Which version of Homebridge/HOOBS do you have?
**4.0.70**

* Which platform do you run Homebridge/HOOBS on (e.g. Raspberry Pi/Windows/HOOBS Box)? Please also mention your version of Node.js/NPM if known.
**Hoobs box
Node version 14.17.1
**

* Which version of this plugin (homebridge-ewelink) do you have? Has the issue started since upgrading from a previous version?

Last version, issue not coming from an update 

* Which eWeLink devices do you have that are causing issues? Please include product models if applicable.

**Zigbee switch**

### Please paste any relevant logs below.

```

eWeLink BridgeERRORRangeError [ERR_OUT_OF_RANGE]: The value of "offset" is out of range. It must be >= 0 and <= 369. Received 9017 at boundsError (internal/buffer.js:83:9) at Uint8Array.readUInt8 (internal/buffer.js:249:5) at DnsSdParser._parseLabel (/var/lib/hoobs/ewelinkbridge/node_modules/node-dns-sd/lib/dns-sd-parser.js:195:23) at DnsSdParser._parseLabel (/var/lib/hoobs/ewelinkbridge/node_modules/node-dns-sd/lib/dns-sd-parser.js:198:22) at DnsSdParser.parse (/var/lib/hoobs/ewelinkbridge/node_modules/node-dns-sd/lib/dns-sd-parser.js:85:21) at DnsSd._receivePacket (/var/lib/hoobs/ewelinkbridge/node_modules/node-dns-sd/lib/dns-sd.js:559:23) at Socket.<anonymous> (/var/lib/hoobs/ewelinkbridge/node_modules/node-dns-sd/lib/dns-sd.js:520:9) at Socket.emit (events.js:375:28) at Socket.emit (domain.js:470:12) at UDP.onMessage (dgram.js:941:8)
eWeLink Bridge starting
eWeLink BridgeLoaded plugin 'homebridge-ewelink'
eWeLink BridgeLoading 1 platforms...
eWeLink BridgeeWeLink[v6.8.0] initialised. Syncing with eWeLink.
eWeLink BridgeBridge is running on port 52826.
eWeLink BridgeeWeLinkLAN monitoring started.
eWeLink BridgeeWeLink[Via Bologna] initialised and LAN mode unavailable as not supported.
eWeLink BridgeeWeLink[Pannello Led] initialised and LAN mode unavailable as not supported.
eWeLink BridgeeWeLink[Lampada] initialised and LAN mode unavailable as not supported.
eWeLink BridgeeWeLink[Via Tollegno] initialised and LAN mode unavailable as not supported.
eWeLink BridgeeWeLink[Plafoniere] initialised and LAN mode unavailable as not supported.
eWeLink BridgeeWeLink[Prato] initialised and LAN mode unavailable as not supported.
eWeLink BridgeeWeLink[Scala] initialised and LAN mode unavailable as not supported.
eWeLink BridgeeWeLink✓ Setup complete. Don't forget to ☆ this plugin on GitHub if you're finding it useful!
eWeLink BridgeERRORRangeError [ERR_OUT_OF_RANGE]: The value of "offset" is out of range. It must be >= 0 and <= 369. Received 9017 at boundsError (internal/buffer.js:83:9) at Uint8Array.readUInt8 (internal/buffer.js:249:5) at DnsSdParser._parseLabel (/var/lib/hoobs/ewelinkbridge/node_modules/node-dns-sd/lib/dns-sd-parser.js:195:23) at DnsSdParser._parseLabel (/var/lib/hoobs/ewelinkbridge/node_modules/node-dns-sd/lib/dns-sd-parser.js:198:22) at DnsSdParser.parse (/var/lib/hoobs/ewelinkbridge/node_modules/node-dns-sd/lib/dns-sd-parser.js:85:21) at DnsSd._receivePacket (/var/lib/hoobs/ewelinkbridge/node_modules/node-dns-sd/lib/dns-sd.js:559:23) at Socket.<anonymous> (/var/lib/hoobs/ewelinkbridge/node_modules/node-dns-sd/lib/dns-sd.js:520:9) at Socket.emit (events.js:375:28) at Socket.emit (domain.js:470:12) at UDP.onMessage (dgram.js:941:8)

```

**Comment by bwp91** [comment]
Hi @Stetocasa 

Are you getting this error every time the plugin starts? Is the hoobs bridge in a constant restarting loop?

**Comment by Stetocasa** [comment]
> Hi @Stetocasa 
> 
> 
> 
> Are you getting this error every time the plugin starts? Is the hoobs bridge in a constant restarting loop?

No, now stopped and all works. The issue happened last night (solved deleting and install again the plug-in) and today after a couple of hours comes back to work

**Comment by bwp91** [comment]
I think this problem will be fixed in an upcoming release.

Since you’re on HOOBS 4, I can’t get you to test a beta version of the plugin.

**Comment by bwp91** [comment]
This should be fixed in v7.0.0 of the plugin
